features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/workflowextensions/default-workflow-extensions.xml

```diff
@@ -0,0 +1,70 @@
+<WorkFlowExtensions>
+    <ApplicationCreation executor="org.wso2.carbon.apimgt.impl.workflow.ApplicationCreationSimpleWorkflowExecutor"/>
+    <!--ApplicationCreation executor="org.wso2.carbon.apimgt.impl.workflow.ApplicationCreationWSWorkflowExecutor">
```

Comment:
lets add our new executors instead of *WSWorkflowExecutor comment it out

Reply:
fixed with e00fe40

Reply:
still WSWorkflowExecutor are here?